src/constants/Colors.ts

```diff
@@ -28,4 +28,6 @@ export default {
   connectionLableColor: '#ffffff',
   graphBarColor: 'grey',
   rootViewColor: '#fff',
+  ventOnButtonColor: 'green',
```

Comment:
can you check the other colours in this page? I chose a different red and green colour in some places, and you may want to re-use those